README.md

```diff
@@ -1 +1,44 @@
-The documentation is on [spilo.readthedocs.org](http://spilo.readthedocs.org/).
+## Spilo: HA PostgreSQL Clusters with Docker
+
+Spilo is a Docker image that provides a resilient, High Available PostgreSQL cluster (HA-cluster) that you can configure and start within minutes. Spilo can run several clusters at one time, with one cluster acting as a master and the others as slaves. Its name derives from სპილო [spiːlɒ], the Georgian word for "elephant." 
+
+Spilo is currently evolving: Its creators are working on a Postgres operator that would make it simpler to deploy scalable Postgres clusters in a Kubernetes environment, and do maintenance tasks. Spilo would serve as an essential building block for this.
+
+Spilo works in combination with [Patroni](https://github.com/zalando/patroni), a template for PostgreSQL HA. Patroni works with [etcd](https://coreos.com/etcd/), [ZooKeeper](https://zookeeper.apache.org/) or [Consul](https://www.consul.io/), and manages the [PostgreSQL](https://www.postgresql.org) databases running in your Spilo cluster. It ensures that there is only one master within your Spilo. (It's worth noting here that Patroni includes a Helm Chart that enables you to deploy a five-node Patroni cluster using a Kubernetes PetSet, in a Google Compute Engine environment.)
```

Comment:
Yeah, the Helm chart is actually named wrongly, since it's deploying Spilo docker images, not just "bare" Patroni.  At some point we tried to avoid using Spilo for anything that is not STUPS, that resulted in the confusion with the names. I'd put the Helm chart mention into the separate chapter (Real-word usage or something like this), where we can mention Helm, Stups (or maybe not) and say some words about upcoming operator development. 

I'd also phrase the Patroni - Helm chart relationship differently, i.e. there is a Helm chart that relies on Spilo and Patroni to provision a five-node PostgreSQL HA cluster.